### PR TITLE
Multiple small changes to hit functions

### PR DESCRIPTION
**Pull request by ak-2485**
Includes: a few indexing errors fixed in angular_hit, changed angular_hit to work for circle not centered at the origin, added additional print stmnts. to angular_hit. Changed line
67 of radial_hit to include exponent. Fixed conditional in angular_hit
for case hit==true.

### CODE REVIEW

File: angular_hit.m [hunk 1]
```diff
@@ -40,45 +40,47 @@
     % Solve the systems Az=b to check for intersection
     Amin = [xmin, -ray_direction(1); ymin, -ray_direction(2)];
```
Comment (cgyurgyik) [inline angular_hit.m]: The math isn't entirely intuitive for me here. What do Amin and Amax represent? If its simple, you can just explain it here. If its complex, I'd add a few more comments for Youhan and I to reference later.

File: polarCoordinateTraversal.m [hunk 2]
```diff
@@ -130,9 +129,10 @@
         % 1. Calculate tMaxR, tMaxTheta
         [is_radial_hit, tMaxR, tStepR] = radial_hit(ray_origin, ray_direction, ...
             current_voxel_ID_r, circle_center, circle_max_radius, delta_radius, verbose);
+        pause
```
Comment (cgyurgyik) [inline polarCoordinateTraversal.m]: Don't need this anymore correct?

File: angular_hit.m (hunk 1 above)
Comment (ak-2485) [reply angular_hit.m]: We are solving the system [rcos(xmin)-circle_center(1) rsin(xmin)-circle_center(2)]' = [u0 u1]' + [v0 v1]'t and same for xmax ymax as args to trig functions. The var array is [r t]; so A[r t] = b.

Comment (ak-2485) [reply angular_hit.m]: Sorry, I don't think that's so clear. I'm trying to make dinner and type at the same time. Will be back around 8 pm.

**Pull request closed without merging**